### PR TITLE
52 form schema and implement submit function for edit and add pages

### PR DESCRIPTION
**Pull request by gitCarrot**
[https://github.com/gameboiss/ask-me/blob/main/.github/pull_request_template.md]: #

# PR Description

- Make pages for add and edit events.
- Set schema for data
- Fixed some eslint error in ts file

# Checklist:

- [X] My code follows the style guidelines of this project
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/imports/ui/pages/AddEvents.jsx [hunk 1]
```diff
@@ -1,11 +1,95 @@
...
+});
 
-const AddEvents = () => (
-  <Container className="py-3">
-    <HelpButton />
-  </Container>
-);
+const bridge = new SimpleSchema2Bridge(formSchema);
 
-export default AddEvents;
```
Comment (jlo2224) [inline app/imports/ui/pages/AddEvents.jsx:11]: Renaming from AddEvents to AddEvent is fine, but we should make it consistent and rename the file as well

File: app/imports/ui/pages/AddEvents.jsx [hunk 2]
```diff
@@ -1,11 +1,95 @@
...
+      'Marketing', 'Accounting', 'Finance', 'Entrepreneurship', 'Biology', 'Chemistry',
+      'Physics', 'Environmental Science', 'Geology', 'Psychology', 'Sociology', 'Political Science',
+      'Economics', 'Anthropology', 'English', 'History', 'Philosophy', 'Religious Studies', 'Classics',
+      'Fine Arts', 'Music', 'Theater', 'Film', 'Creative Writing', 'Nursing', 'Pre-Med', 'Public Health',
+      'Health Sciences', 'Physical Therapy', 'Linguistics', 'Journalism', 'Advertising', 'Public Relations',
+      'Communication Studies', 'Law', 'Criminal Justice', 'Paralegal Studies', 'Political Science', 'Sociology',
+      'International Relations', 'Global Studies', 'Foreign Languages', 'Engineering'].sort(),
+    defaultValue: 'Accounting',
+  },
+  companyId: {
```
Comment (jlo2224) [inline app/imports/ui/pages/AddEvents.jsx]: For companyId and forms in general, if the user cannot manually input the data, we should only include the data in the submit function and not the formSchema. Just for good practice

File: app/imports/ui/pages/AddEvents.jsx [hunk 3]
```diff
@@ -1,11 +1,95 @@
...
+    type: String,
+    defaultValue: Companies.companyPublicationName,
+  },
+  createdAt: {
```
Comment (jlo2224) [inline app/imports/ui/pages/AddEvents.jsx]: Similar as above with companyId

File: app/imports/ui/pages/EditEvent.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+/* Renders the EditEvent page for editing a single document. */
+const EditEvent = () => {
+  // Get the documentID from the URL field. See imports/ui/layouts/App.jsx for the route containing :_id.
+  const { _id } = useParams();
+  // console.log('EditEvent', _id);
+  // useTracker connects Meteor data to React components. https://guide.meteor.com/react.html#using-withTracker
+  const { doc, ready } = useTracker(() => {
+    // Get access to Contact documents.
+    const subscription = Meteor.subscribe(Events.adminPublicationName);
+    const subscription2 = Meteor.subscribe(Events.companyPublicationName);
```
Comment (jlo2224) [inline app/imports/ui/pages/EditEvent.jsx:23]: I'd recommend using different variable names here

File: app/imports/ui/pages/AddEvents.jsx [hunk 5]
```diff
@@ -1,11 +1,95 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
-import HelpButton from '../components/HelpButton';
```
Comment (jlo2224) [inline app/imports/ui/pages/AddEvents.jsx:3]: I'd like to keep HelpButton included. You can just put it as the last component before closing the container with </Container>

File: app/imports/ui/pages/ListEvents.jsx [hunk 6]
```diff
@@ -1,11 +1,52 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
-import HelpButton from '../components/HelpButton';
```
Comment (jlo2224) [inline app/imports/ui/pages/ListEvents.jsx:3]: Same as before, I'd like to keep HelpButton included. You can just put it as the last component before closing the container with </Container>

**Review by jlo2224 — CHANGES_REQUESTED** [review]
Mostly looks good, just a few stylistic changes. Also just as a general comment, instead of using AddEvents page and ListEvents page, we can make a separate "Events" page. But this should be fine, we could just store Events.jsx and have it load both of them in separate Col xs={6}

File: app/imports/ui/pages/EditEvent.jsx (hunk 4 above)
Comment (gitCarrot) [reply app/imports/ui/pages/EditEvent.jsx:23]: What do you mean 'different variable names here'?

**Review by Prayag-Das — APPROVED** [review]

**Pull request merged**